### PR TITLE
test with pre-installed libpcsclite1 package

### PR DESCRIPTION
**Pull request by developer-guy**
Signed-off-by: Batuhan Apaydın <batuhan.apaydin@trendyol.com>

based on the conversation in PR #15 
https://github.com/sigstore/cosign-installer/pull/15#pullrequestreview-732782575

### CODE REVIEW

**Comment by developer-guy** [comment]
@hectorj2f WDYT, can you please take another look? 🙏

File: action.yml [hunk 1]
```diff
@@ -47,7 +47,15 @@ runs:
...
             # v0.6.0's linux release has a dependency on `libpcsclite1`
             sudo apt-get update -q
-            sudo apt-get install -yq libpcsclite1
+            set +e
+            sudo dpkg -s libpcsclite1
+            if [ $? -eq 0 ]; then
+                echo "INFO: libpcsclite1 package is already installed"
+            else
+                 echo "INFO: libpcsclite1 package is not installed, installing it now."
+                 sudo apt-get install -yq libpcsclite1
```
Comment (cpanato) [inline action.yml:56]: should we do the spt update here as well?

File: action.yml [hunk 2]
```diff
@@ -47,7 +47,15 @@ runs:
           if [[ ${{ inputs.cosign-release }} == 'v0.6.0' ]]; then
             # v0.6.0's linux release has a dependency on `libpcsclite1`
             sudo apt-get update -q
```
Comment (hectorj2f) [inline action.yml:49]: Question: do we need to run the update even if the dependency exist ? That is my main concern

Comment (cpanato) [reply action.yml:49]: if the dependency exist i don't think we need to update

Comment (developer-guy) [reply action.yml]: yes you are right, I fixed it.

File: action.yml (hunk 1 above)
Comment (developer-guy) [reply action.yml:56]: of course, it makes sense, thank you @cpanato 🙏

File: action.yml (hunk 2 above)
Comment (developer-guy) [reply action.yml]: thank you so much, folks @cpanato @hectorj2f

**Review by hectorj2f — APPROVED** [review]

**Review by cpanato — APPROVED** [review]

**Pull request merged**